pkg/virt-launcher/virtwrap/agent-poller/agent_poller_test.go

```diff
@@ -117,5 +117,19 @@ var _ = Describe("Qemu agent poller", func() {
 			Eventually(func() int { return execCounter }, 3*time.Second, time.Second).Should(Equal(3))
 		})
 
+		It("executes the agent commands in increasing intervals at initial run", func() {
+			w := PollerWorker{
+				CallTick:      30,
+				AgentCommands: []AgentCommand{fakeAgentCommandName},
+			}
+			c := make(chan struct{})
+
+			var execCounter int
+			go w.Poll(func(commands []AgentCommand) { execCounter++ }, c)
+			time.Sleep(pollInitialInterval + time.Millisecond)
```

Comment:
same comment as in the 2nd test

Reply:
Same answer.

Reply:
Ref
https://github.com/kubevirt/kubevirt/pull/4512#discussion_r521143176